app/models/GatewayAccount.class.js

```diff
@@ -1,14 +1,16 @@
-const GatewayAccountCredential = require('@models/GatewayAccountCredential.class')
+const { GatewayAccountCredential, CREDENTIAL_STATE } = require('@models/GatewayAccountCredential.class')
 
 /**
  * @class GatewayAccount
  * @property {string} name - The name of the gateway account
  * @property {string} id - The id of the gateway account
  * @property {string} type - The type of the gateway account (e.g. test/live)
  * @property {string} description - The description of the gateway account
+ * @property {boolean} allowMoto - whether MOTO payments are enabled on the gateway account
  * @property {string} analyticsId - Google analyticsId of the gateway account
  * @property {boolean} toggle3ds - whether 3DS is enabled or not on this gateway account
  * @property {[GatewayAccountCredential]} gatewayAccountCredentials - available credentials for gateway account
+ * @property {GatewayAccountCredential} activeCredential - the active credential for the gateway account
```

Comment:
this is an optional property so should be wrapped in square brackets, see `gatewayAccountCredentials` above

Reply:
Are you sure? I thought `[GatewayAccountCredential]` means a list of credentials, which a gateway account does have.

Reply:
sorry I meant `@property {GatewayAccountCredential} [activeCredential]`. My B!